src/Builders/RouteValueDictionaryBuilder.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Generic;
```

Comment:
Is this using needed if the rest of the file hasn't changed?

Reply:
Visual Studio doing some sneaky stuff. Removed